packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -342,6 +343,15 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
         validatePrismaClientOptions(optionsArg, config.datasourceNames)
       }
 
+      const logEmitter = new EventEmitter().on('error', (e) => {
```

Comment:
In `eventemitter3`

> We do not throw an error when you emit an error event and nobody is listening.

Reply:
Agreed that's one of the reasons why we should use eventemitter3 altogether and stop depending on Node's implementation.